apis/v1beta1/shared_types.go

```diff
@@ -610,6 +610,11 @@ type AddressType string
 // +k8s:deepcopy-gen=false
 type HeaderName string
 
+// A duration is a ...
+//
+// +kubebuilder:validation:Pattern=`^([0-9]{1,5}(h|m|s|ms)){1,4}$`
+type Duration string
```

Comment:
Do we also need to define this in `v1alpha2` as a clone of the `v1beta1` type, as is done for other shared types? For example https://github.com/kubernetes-sigs/gateway-api/blob/e32217ea84af9d69817f835887949f005a9e1e2c/apis/v1alpha2/shared_types.go#L176

^^^^ @robscott, please advise

Reply:
addressed in 6281ef19931f78d873d920088ba4abb36055f346